stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -1,19 +1,23 @@
 {
     "connection": {
         "type": {
-            "displayName": "IBM® Guardium"
```

Comment:
I think this was correct before the change, at least based on looking at the config files for other connectors. @mdazam1942 am I right about this?

Reply:
I revert this change